pubspec.yaml

```diff
@@ -12,10 +12,10 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  web: ">=0.5.0 <2.0.0"
```

Comment:
This will make a ton of people complain about incompatibility with other packages.